src/components/Admin/ListUser/ListUser.js

```diff
@@ -255,6 +283,18 @@ export default class ListUser extends Component {
         let userList = response.users;
         let users = [];
         userList.map((data, i) => {
+          let devices = [];
+          let keys = Object.keys(data.devices);
+          keys.forEach(j => {
```

Comment:
Name variables correctly. `i` and `j` are not clear

Reply:
`i` and `j` are indices commonly used during iterations.